### PR TITLE
Updated with new Simplified Chinese string translation

### PR DESCRIPTION
**Pull request by sbzlzh**
@TheOnly8Z Could you look at it when you have time, very little translation :) thank you!

### CODE REVIEW

File: lua/terrortown/lang/zh_hans.lua [hunk 1]
```diff
@@ -2059,48 +2059,48 @@ L.label_keyhelper_weapon_drop = "尽可能丢出所选武器"
...
 -- 2023-12-14
---L.marker_vision_owner = "Owner: {owner}"
---L.marker_vision_distance = "Distance: {distance}m"
---L.marker_vision_visible_for_0 = "Visible for: you"
---L.marker_vision_visible_for_1 = "Visible for: team"
---L.marker_vision_visible_for_2 = "Visible for: everyone"
---L.marker_vision_distance_collapsed = "{distance}m"
+L.marker_vision_owner = "所有者: {owner}"
+L.marker_vision_distance = "距离: {distance}米"
+L.marker_vision_visible_for_0 = "可见度: 您"
```
Comment (TheOnly8Z) [inline lua/terrortown/lang/zh_hans.lua:2090]:
```suggestion
L.marker_vision_visible_for_0 = "可见对象: 您"
```

File: lua/terrortown/lang/zh_hans.lua [hunk 2]
```diff
@@ -2059,48 +2059,48 @@ L.label_keyhelper_weapon_drop = "尽可能丢出所选武器"
...
+L.marker_vision_visible_for_1 = "可见度: 团队"
```
Comment (TheOnly8Z) [inline lua/terrortown/lang/zh_hans.lua:2091]:
```suggestion
L.marker_vision_visible_for_1 = "可见对象: 团队"
```

File: lua/terrortown/lang/zh_hans.lua [hunk 3]
```diff
@@ -2059,48 +2059,48 @@ L.label_keyhelper_weapon_drop = "尽可能丢出所选武器"
...
+L.marker_vision_visible_for_2 = "可见度: 所有人"
```
Comment (TheOnly8Z) [inline lua/terrortown/lang/zh_hans.lua:2092]:
```suggestion
L.marker_vision_visible_for_2 = "可见对象: 所有人"
```

File: lua/terrortown/lang/zh_hans.lua [hunk 4]
```diff
@@ -2175,12 +2175,12 @@ L.help_crosshair_scale_enable = [[
...
 -- 2024-01-25
---L.header_roles_magnetostick = "Magneto Stick"
---L.label_roles_ragdoll_pinning = "Enable ragdoll pinning"
---L.magneto_stick_help_carry_rag_pin = "Pin ragdoll"
---L.magneto_stick_help_carry_rag_drop = "Drop ragdoll"
---L.magneto_stick_help_carry_prop_release = "Release prop"
---L.magneto_stick_help_carry_prop_drop = "Drop prop"
+L.header_roles_magnetostick = "磁力棒"
+L.label_roles_ragdoll_pinning = "启用布娃娃固定"
+L.magneto_stick_help_carry_rag_pin = "针形布娃娃"
```
Comment (TheOnly8Z) [inline lua/terrortown/lang/zh_hans.lua:2183]:
```suggestion
L.magneto_stick_help_carry_rag_pin = "固定布娃娃"
```

**Review by TheOnly8Z — CHANGES_REQUESTED** [review]

**Comment by sbzlzh** [comment]
Is there anything else.

**Comment by sbzlzh** [comment]
@TheOnly8Z Please take a look at it. It's a little different.

**Comment by sbzlzh** [comment]
@TimGoll How do you feel about removing the extra spaces and switching positions.

**Comment by TimGoll** [comment]
> @TimGoll How do you feel about removing the extra spaces and switching positions.

yep, that's fine, thank you!

**Comment by ZenBre4ker** [comment]
If @TheOnly8Z approves, we will approve as well.

File: lua/terrortown/lang/en.lua [hunk 5]
```diff
@@ -2189,5 +2176,15 @@ L.magneto_stick_help_carry_rag_drop = "Drop ragdoll"
...
+-- 2024-01-27
+L.decoy_help_primary = "Throw Decoy on the ground"
+L.decoy_help_secondary = "Stick Decoy to surface"
+
+--2024-01-29
+L.marker_vision_visible_for_0 = "Visible for you"
+L.marker_vision_visible_for_1 = "Visible for your role"
+L.marker_vision_visible_for_2 = "Visible for your team"
+L.marker_vision_visible_for_3 = "Visible for everyone"
+
```
Comment (ZenBre4ker) [inline lua/terrortown/lang/en.lua:2188]: Oh wait, this shoulnt do any changes in en. Probably something wrong with your rebase?

File: lua/terrortown/lang/zh_hans.lua [hunk 6]
```diff
@@ -2168,26 +2165,26 @@ L.help_crosshair_scale_enable = [[
...
---L.magneto_stick_help_carry_prop_drop = "Drop prop"
+--2024-01-29
+L.marker_vision_visible_for_0 = "您可见"
+L.marker_vision_visible_for_1 = "您的角色可见"
+L.marker_vision_visible_for_2 = "您的团队可见"
+L.marker_vision_visible_for_3 = "每个人都可见"
 
 -- 2024-02-14
---L.throw_no_room = "You have no space here to throw this device"
+L.throw_no_room = "这里没地方扔这个装置"
```
Comment (ZenBre4ker) [inline lua/terrortown/lang/zh_hans.lua:2190]: Same here, something wrong with the order of your changes.

**Review by ZenBre4ker — CHANGES_REQUESTED** [review]

File: lua/terrortown/lang/en.lua (hunk 5 above)
Comment (TimGoll) [reply lua/terrortown/lang/en.lua:2188]: nah, take a look at the dates, they were previously in the wrong order, now they aren't anymore. It is not important, but not a problem either. It is fine with me

**Review by ZenBre4ker — APPROVED** [review]

**Pull request closed without merging**